packages/generator/src/generateClientSdkPackage.ts

```diff
@@ -0,0 +1,204 @@
+    path.join(baseOutDir, "ontology", "objects.js"),
+    `module.exports = require("../../dist/module/ontology/objects")`,
+  );
+  await minimalFs.writeFile(
+    path.join(baseOutDir, "ontology", "objects.d.ts"),
+    `export * from "../dist/module/ontology/objects"`,
+  );
+}
+
+function getTsCompilerOptions(packageType: "commonjs" | "module") {
+  const commonTsconfig = {
+    importHelpers: true,
+
+    declaration: true,
+
+    isolatedModules: true,
+    esModuleInterop: true,
+
+    forceConsistentCasingInFileNames: true,
+    strict: true,
+
+    skipLibCheck: true,
+  };
+
+  const compilerOptions = packageType === "commonjs"
+    ? {
+      ...commonTsconfig,
+      module: "commonjs",
+      target: "es2018",
+    }
+    : {
+      ...commonTsconfig,
+      module: "NodeNext",
+      target: "ES2020",
+    };
+  return compilerOptions;
+}
+
+export interface DependencyVersions {
+  typescriptVersion: string;
+  tslibVersion: string;
+  areTheTypesWrongVersion: string;
+  osdkApiVersion: string;
+  osdkClientVersion: string;
+  osdkLegacyClientVersion: string;
+}
+
+export function getPackageJsonContents(
+  name: string,
+  version: string,
+  sdkVersion: "2.0" | "1.1",
+  {
+    typescriptVersion,
+    tslibVersion,
+    areTheTypesWrongVersion,
+    osdkApiVersion,
+    osdkClientVersion,
+    osdkLegacyClientVersion,
+  }: DependencyVersions,
+) {
+  const esmPrefix = "./dist/module";
+  const commonjsPrefix = "./dist/commonjs";
+  return {
+    name,
+    version,
+    main: `${commonjsPrefix}/index.js`,
+    module: `${esmPrefix}/index.js`,
+    exports: {
+      ".": {
+        import: `${esmPrefix}/index.js`,
+        require: `${commonjsPrefix}/index.js`,
+      },
+      "./ontology/objects": {
+        import: `${esmPrefix}/ontology/objects${
+          sdkVersion === "2.0" ? "" : "/index"
+        }.js`,
+        require: `${commonjsPrefix}/ontology/objects${
+          sdkVersion === "2.0" ? "" : "/index"
+        }.js`,
+      },
+    },
+    scripts: {
+      prepack:
+        `tsc -p ${esmPrefix}/tsconfig.json && tsc -p ${commonjsPrefix}/tsconfig.json`,
+      check: "npm exec attw $(npm pack)",
+    },
+    devDependencies: {
+      "typescript": typescriptVersion,
+      "tslib": tslibVersion,
+      "@arethetypeswrong/cli": areTheTypesWrongVersion,
+      "@osdk/api": osdkApiVersion,
+      ...(sdkVersion
+          === "2.0"
+        ? { "@osdk/client": osdkClientVersion }
+        : { "@osdk/legacy-client": osdkLegacyClientVersion }),
+    },
+    peerDependencies: {
+      "@osdk/api": osdkApiVersion,
+      ...(sdkVersion === "2.0"
+        ? { "@osdk/client": osdkClientVersion }
+        : { "@osdk/legacy-client": osdkLegacyClientVersion }),
+    },
+    files: [
+      "**/*.js",
+      "**/*.d.ts",
+      "dist/**/package.json",
+    ],
+  };
+}
+
+async function writeJson(
+  minimalFs: MinimalFs,
+  filePath: string,
+  body: unknown,
+) {
+  // consola.info(`Writing ${filePath}`);
```

Comment:
nit: do we want to get rid of these?

Reply:
This was only put in here for me for the hackweek we had 2 weeks ago